Examples/Scripts/Python/seeding.py

```diff
@@ -1,13 +1,11 @@
-#!/usr/bin/env python3
```

Comment:
I guess the usual `#!/usr/bin/env python3` was accidentally removed. Can we have it back?

Reply:
My (or black's) fault. Sorry. Will restore in the next push (keep forgetting about the black)